### PR TITLE
Fixes #21378 - Add CSV export on tasks index

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: app/helpers/foreman_tasks/tasks_helper.rb [hunk 1]
```diff
@@ -1,17 +1,7 @@
 module ForemanTasks
   module TasksHelper
-    def format_task_input(task, include_action = false)
```
Comment (tbrisker) [inline app/helpers/foreman_tasks/tasks_helper.rb]: include_action is never set to false, removed it completely.

**Comment by iNecas** [comment]
Needs rebase

**Comment by tbrisker** [comment]
@iNecas thanks, rebased.

**Comment by tbrisker** [comment]
@iNecas @adamruzicka ping?

**Comment by adamruzicka** [comment]
@tbrisker thanks and sorry for the delay

**Pull request merged**